aspnetcore/tutorials/min-web-api/samples/7.x/todoGroup/Program.cs

```diff
@@ -0,0 +1,60 @@
+// <snippet_all>
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDbContext<TodoDb>(opt => opt.UseInMemoryDatabase("TodoList"));
+builder.Services.AddDatabaseDeveloperPageExceptionFilter();
+var app = builder.Build();
+
+// <snippet_group>
+RouteGroupBuilder group = app.MapGroup("/todoitems");
```

Comment:
```suggestion
var todoItems = app.MapGroup("/todoitems");
```
I think `group` is a LINQ keyword so the syntax highlighting ends up being weird here. I think it's better to name the group based on the prefix since it makes it easier to find groups in user code.